vellum/workflows/triage_message/nodes/parse_function_call_node.py

```diff
@@ -4,35 +4,62 @@
 from .triage_message_node import TriageMessageNode
 
 
+def _is_function_call() -> LazyReference:
```

Comment:
This is a good idea. Add a name:str argument to this helper so that we can add the function call name check here and centralize most of the ports below